o.apache.tools.ant.module/src/org/apache/tools/ant/module/resources/Project.xml_

```diff
@@ -1,4 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
This license header is new - it will be copied into the user file verbatim. Please add an exclusion for this file in the rat report with a comment along the line "user visible template file" and remove the license header.

Reply:
Cool, Ill address this tomorrow